### PR TITLE
front: rem to px in /common

### LINKED ISSUE #9841: front: convert rem to px in /common
We need to convert all the sizes from rem to px.

Convert rem to pix in all the files in /common

### PR DESCRIPTION
**Pull request by clarani**
closes #9841

### CODE REVIEW

File: front/src/modules/conflict/styles/_conflictList.scss [hunk 1]
```diff
@@ -28,86 +28,86 @@
...
-        gap: 0.5rem;
+        padding: 16px 0 16px 16px;
+        gap: 8px;
 
         .train-name-card {
           font-family: 'IBM Plex Mono', monospace;
-          height: 1.5rem;
-          max-width: 13.0437rem;
+          height: 24px;
+          max-width: 208.6992px;
```
Comment (theocrsb) [inline front/src/modules/conflict/styles/_conflictList.scss:99]: Maybe we should round off these values?

File: front/src/styles/scss/applications/operationalStudies/_home.scss [hunk 2]
```diff
@@ -2,8 +2,8 @@
   display: flex;
   align-items: center;
   flex-wrap: wrap;
-  gap: 1rem;
-  padding: 1rem 0 1rem;
+  gap: 16px;
+  padding: 16px 0 16px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_home.scss]:
```suggestion
  padding: 16px 0;
```

File: front/src/styles/scss/applications/operationalStudies/_project.scss [hunk 3]
```diff
@@ -172,18 +172,18 @@
     display: flex;
     align-items: center;
     flex-wrap: wrap;
-    gap: 1rem;
-    padding: 1rem 0.5rem 1rem 0.5rem;
-    margin-top: 2rem;
+    gap: 16px;
+    padding: 16px 8px 16px 8px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_project.scss]:
```suggestion
    padding: 16px 8px;
```

File: front/src/styles/scss/applications/operationalStudies/_study.scss [hunk 4]
```diff
@@ -260,18 +260,18 @@
     display: flex;
     align-items: center;
     flex-wrap: wrap;
-    gap: 1rem;
-    padding: 1rem 0.5rem 1rem 0.5rem;
-    margin-top: 2rem;
+    gap: 16px;
+    padding: 16px 8px 16px 8px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_study.scss]:
```suggestion
    padding: 16px 8px;
```

File: front/src/styles/scss/applications/operationalStudies/_project.scss [hunk 5]
```diff
@@ -172,18 +172,18 @@
...
   .studies-list {
     overflow: auto;
     max-height: 50vh;
-    margin-left: -0.5rem;
-    margin-right: -0.5rem;
-    padding-top: 1rem;
-    padding-left: 0.5rem;
-    padding-right: 0.5rem;
+    margin-left: -8px;
+    margin-right: -8px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_project.scss]:
```suggestion
    margin-inline: -8px;
```

File: front/src/styles/scss/applications/operationalStudies/_project.scss [hunk 6]
```diff
@@ -172,18 +172,18 @@
...
+    padding-top: 16px;
+    padding-left: 8px;
+    padding-right: 8px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_project.scss]:
```suggestion
    padding: 16px 8px 0;
```

File: front/src/styles/scss/applications/operationalStudies/_study.scss [hunk 7]
```diff
@@ -260,18 +260,18 @@
...
   .scenarios-list {
     overflow: auto;
     max-height: 50vh;
-    margin-left: -0.5rem;
-    margin-right: -0.5rem;
-    padding-top: 1rem;
-    padding-left: 0.5rem;
-    padding-right: 0.5rem;
+    margin-left: -8px;
+    margin-right: -8px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_study.scss]:
```suggestion
    margin-inline: -8px;
```

File: front/src/styles/scss/applications/operationalStudies/_study.scss [hunk 8]
```diff
@@ -260,18 +260,18 @@
...
+    padding-top: 16px;
+    padding-left: 8px;
+    padding-right: 8px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/operationalStudies/_study.scss]:
```suggestion
    padding: 16px 8px 0;
```

File: front/src/styles/scss/applications/rollingStockEditor/_rollingStockForm.scss [hunk 9]
```diff
@@ -303,12 +303,12 @@
...
-    column-gap: 2.1rem;
+    row-gap: 24px;
+    column-gap: 33.6px;
     padding-top: 0;
-    padding-left: 1rem;
-    padding-right: 2rem;
-    padding-bottom: 1rem;
+    padding-left: 16px;
+    padding-right: 32px;
+    padding-bottom: 16px;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/rollingStockEditor/_rollingStockForm.scss]:
```suggestion
    padding: 0 32px 16px 16px;
```

File: front/src/styles/scss/applications/stdcm/_header.scss [hunk 10]
```diff
@@ -19,21 +19,21 @@
...
   .stdcm-header__notification {
-    max-height: 3rem;
+    max-height: 48px;
     border-radius: 2px;
     border: 4px solid rgba(112, 193, 229, 1);
     background-color: rgba(230, 247, 255, 1);
     padding: 7px 16px 9px 16px;
     color: rgba(33, 100, 130, 1);
-    font-size: 16px;
+    font-size: 1rem;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/stdcm/_header.scss:28]:
```suggestion
    font-size: 16px;
```

File: front/src/styles/scss/applications/stdcm/_header.scss [hunk 11]
```diff
@@ -19,21 +19,21 @@
...
     font-weight: 600;
     font-style: SemiBold;
     letter-spacing: 0px;
     line-height: 24px;
 
     a {
       color: rgba(24, 68, 239, 1);
-      font-size: 16px;
+      font-size: 1rem;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/stdcm/_header.scss:36]:
```suggestion
      font-size: 16px;
```

File: front/src/styles/scss/applications/stdcm/_home.scss [hunk 12]
```diff
@@ -26,7 +26,7 @@
         padding: 8px 51px 8px 0;
         .scenario-explorator-card,
         .scenario-explorator-card-noscenario {
-          font-size: 20px;
+          font-size: 1.25rem;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/stdcm/_home.scss:29]:
```suggestion
          font-size: 20px;
```

File: front/src/styles/scss/applications/stdcm/_home.scss [hunk 13]
```diff
@@ -214,7 +214,7 @@
         justify-content: center;
         color: rgb(11, 114, 60);
         padding-top: 23px;
-        font-size: 24px;
+        font-size: 1.5rem;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/stdcm/_home.scss:217]:
```suggestion
        font-size: 24px;
```

File: front/src/styles/scss/applications/stdcm/_results.scss
Comment (theocrsb) [inline front/src/styles/scss/applications/stdcm/_results.scss:1]: This file still contains rem

File: front/src/styles/scss/common/_rc-slider.scss
Comment (theocrsb) [inline front/src/styles/scss/common/_rc-slider.scss:1]: Same here

File: front/src/styles/scss/common/components/_tabs.scss [hunk 14]
```diff
@@ -73,7 +73,7 @@
         align-items: center;
         font-weight: 500;
         transition: all ease 0.3s;
-        padding: 0.5rem 1rem 0.5rem;
+        padding: 8px 16px 8px;
```
Comment (theocrsb) [inline front/src/styles/scss/common/components/_tabs.scss]:
```suggestion
        padding: 8px 16px;
```

File: front/src/styles/scss/common/components/_stationCard.scss [hunk 15]
```diff
@@ -44,11 +44,11 @@
     display: flex;
     align-items: baseline;
     position: relative;
-    padding: 0 0.25rem 0 2.5rem;
-    height: 1rem;
-    line-height: 0.8rem;
+    padding: 0 4px 0 40px;
```
Comment (theocrsb) [inline front/src/styles/scss/common/components/_stationCard.scss]:
```suggestion
    padding: 4px 40px;
```

File: front/src/styles/scss/applications/stdcm/_results.scss
Comment (clarani) [reply front/src/styles/scss/applications/stdcm/_results.scss:1]: Yes but I think it's for the font-size, hum it should be ok ?

File: front/src/styles/scss/common/_rc-slider.scss
Comment (clarani) [reply front/src/styles/scss/common/_rc-slider.scss:1]: Same here, the rem are only used for the font-size

File: front/src/modules/conflict/styles/_conflictList.scss (hunk 1 above)
Comment (clarani) [reply front/src/modules/conflict/styles/_conflictList.scss:99]: We will do this in a future PR, as discussed this morning :+1:

File: front/src/styles/scss/applications/operationalStudies/_home.scss (hunk 2 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_home.scss]: Done

File: front/src/styles/scss/applications/operationalStudies/_project.scss (hunk 3 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_project.scss]: Done

File: front/src/styles/scss/applications/operationalStudies/_study.scss (hunk 4 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_study.scss]: Done

File: front/src/styles/scss/applications/operationalStudies/_project.scss (hunk 6 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_project.scss]: Done

File: front/src/styles/scss/applications/operationalStudies/_project.scss (hunk 5 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_project.scss]: Done

File: front/src/styles/scss/applications/operationalStudies/_study.scss (hunk 7 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_study.scss]: Done

File: front/src/styles/scss/applications/operationalStudies/_study.scss (hunk 8 above)
Comment (clarani) [reply front/src/styles/scss/applications/operationalStudies/_study.scss]: Done

File: front/src/styles/scss/applications/rollingStockEditor/_rollingStockForm.scss (hunk 9 above)
Comment (clarani) [reply front/src/styles/scss/applications/rollingStockEditor/_rollingStockForm.scss]: Done

File: front/src/styles/scss/applications/stdcm/_header.scss (hunk 10 above)
Comment (clarani) [reply front/src/styles/scss/applications/stdcm/_header.scss:28]: The font sizes should stay in rem :)

File: front/src/styles/scss/common/components/_tabs.scss (hunk 14 above)
Comment (clarani) [reply front/src/styles/scss/common/components/_tabs.scss]: Done

File: front/src/styles/scss/applications/stdcm/_results.scss
Comment (theocrsb) [reply front/src/styles/scss/applications/stdcm/_results.scss:1]: Yes mb

**Review by theocrsb — APPROVED** [review]
LGTM

**Review by Math-R — APPROVED** [review]
lgtm :)

**Pull request merged**